doc/sphinx-guides/source/user/dataverse-management.rst

```diff
@@ -5,40 +5,40 @@ A dataverse is a container for datasets (research data, code, documentation, and
 
 |image1|
 
-Once a user creates a dataverse they, by default, become the
-administrator of that dataverse. The dataverse administrator has access
-to manage the settings described in this guide.
+Once a user creates a dataverse they, by default, become the administrator of that dataverse. The dataverse administrator has access to manage the settings described in this guide.
 
 .. contents:: |toctitle|
   :local:
 
-Create a Dataverse (Within the "Root" Dataverse)
-===================================================
+Create a New Dataverse
+=======================
 
-Creating a dataverse is easy but first you must be a registered user (see :doc:`/user/account`).
+Creating a dataverse is easy but first you must be a registered user (see :doc:`/user/account`). Once you are logged in, click on the "Add Data" button, or "Add Data" navbar link and in the dropdown menu select "New Dataverse".
 
-#. Once you are logged in click on the "Add Data" button and in the dropdown menu select "New Dataverse".
-#. Once on the "New Dataverse" page fill in the following fields:
-    * **Name**: Enter the name of your dataverse.
+#. **Fill in the following fields on the "New Dataverse" page** (required fields in the form are denoted by a red asterisk, next to the label):
+    * **Dataverse Name**: Enter the name of your dataverse.
+    * **Affiliation**: Add any affiliation that can be associated with this particular dataverse (e.g., project name, institute name, department name, journal name, etc). This is automatically filled out if you have added an affiliation for your user account.
     * **Identifier**: This is an abbreviation, usually lower-case, that becomes part of the URL for the new dataverse. Special characters (~,\`, !, @, #, $, %, ^, &, and \*) and spaces are not allowed. **Note**: if you change this field in the future, the URL for your Dataverse will change (http//.../'identifier'), which will break older links to the page.
+    * **Host Dataverse**: Select a dataverse you would like to create the new dataverse in, by default the root dataverse is selected if you click the Add Data button from the root dataverse, your My Data pg, the Add Data link in the navbar, or the Add a dataset button on the custom homepage (if configured), but you can select any dataverse for which you the proper permissions to create dataverses.
```

Comment:
Should this now be higher since it was moved on the page?

Reply:
Great point. Done.